qcfractal/storage_sockets/models.py

```diff
@@ -257,19 +256,20 @@ class TaskQueue(db.DynamicDocument):
                             # choices=['RUNNING', 'WAITING', 'ERROR', 'COMPLETE'])
     manager = db.StringField(default=None)
 
-
     created_on = db.DateTimeField(required=True, default=datetime.datetime.now)
     modified_on = db.DateTimeField(required=True, default=datetime.datetime.now)
 
-    base_result = db.GenericLazyReferenceField(dbref=True)  # GenericLazyReferenceField()  # can reference Results or any Procedure
+    # can reference Results or any Procedure
+    base_result = db.GenericLazyReferenceField(dbref=True)
 
     meta = {
         'indexes': [
-            '-created_on',
+            'created_on',
             'status',
             'manager',
             # {'fields': ("status", "tag", "hash_index"), 'unique': False}
-            {'fields': ("base_result",), 'unique': True}  # new
+            {'fields': ('base_result',), 'unique': True},  # new
```

Comment:
Is this strictly true as this might reference several tables?

Reply:
Yes, because it's a DBref, i.e., both the table name and the ID.